web/background_worker.py

```diff
@@ -1,8 +1,7 @@
-import os
-import rollbar
-from celery.signals import task_failure
+import traceback
+from celery.signals import task_failure, task_internal_error
+# from util.slack import notify_slack
```

Comment:
Let's delete this commented-out line. When I start to work through the codebase to decouple external services, I'll get rid of all slack-related code anyway.

Reply:
Also delete the two lines where `notify_slack()` is called

Reply:
Ah great, I was wondering if I can just get rid of this, will do!